server/instrumentation-backend/src/main/java/sh/calaba/instrumentationbackend/actions/text/WebViewInputScripts.java

```diff
@@ -188,4 +185,35 @@
             "  }\n" +
             "}\n" +
             "selectText(%1$s,%2$s);";
+
+    /**
+     * Allows to add text according to caret position and selection
+     * @param text text to input
+     * @return script to enter text into active WebView field
+     */
+    public static String enterTextScript(String text) {
+        return String.format(enterScript, text.replaceAll("\'", "\\\\x27"));
```

Comment:
Not entirely sure I understand why `x27` is needed instead of `'`? Your intent is to replace `\'` with `\\'` or am I reading correctly? I would think the replacement `'` ==> `\'` would be the right one? The current solution wouldn't handle the string `"Mikkel's app"` correctly?

Reply:
It will handle it correctly. Result will be like this:
`"Mikkel\x27s app"`. Inside html element this will look correctly: `"Mikkel's app"`
http://jsfiddle.net/m8texkzy/2/
This is safety replacement for HTML.

Reply:
http://rextester.com/XCTRBW60002